src/rantanplan/core.py

```diff
@@ -2,7 +2,12 @@
 # Based on previous work done by Rafael C. Carrasco, José A. Mañas (Communications of the ACM 30(7), 1987)
 # and Javier Sober
 # https://github.com/postdataproject/skas-archived/blob/devel/skas/phonmet/syll/grapheme2syllable.py
-
+#
+# Presyllabification and syllabification rules are taken from Antonio Ríos Mestre's
+# 'El Diccionario Electrónico Fonético del Español'
+# https://www.raco.cat/index.php/Elies/article/view/194843
+# http://elies.rediris.es/elies4/Fon2.htm
+# http://elies.rediris.es/elies4/Fon8.htm
```

Comment:
For God's sake, let's scarp these websites or find if they'e archived!

Reply:
They are on wayback machine ;)
https://web.archive.org/web/2018*/http://elies.rediris.es/elies4/Fon2.htm
https://web.archive.org/web/2018*/http://elies.rediris.es/elies4/Fon8.htm

Reply:
:relieved: